packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -1386,25 +1433,65 @@ export const components: ThemeOptions["components"] = {
   MuiRadio: {
     defaultProps: {
       size: "small",
+      icon: <></>,
+      checkedIcon: <></>,
     },
     styleOverrides: {
       root: ({ theme }) => ({
-        "&:hover": {
+        width: theme.spacing(4),
+        height: theme.spacing(4),
+        borderRadius: theme.spacing(4),
+        borderWidth: "1px",
+        borderStyle: "solid",
+        borderColor: theme.palette.grey[500],
+
+        ".MuiFormControlLabel-root:hover > &": {
+          backgroundColor: "transparent",
+          borderColor: theme.palette.grey[900],
+        },
+        ".Mui-error:hover > &": {
           backgroundColor: "transparent",
+          borderColor: theme.palette.error.dark,
         },
         padding: 0,
         ".Mui-error > &": {
-          color: theme.palette.error.main,
-          "&:hover": {
-            color: theme.palette.error.dark,
+          borderColor: theme.palette.error.main,
+          "&::before": {
+            backgroundColor: theme.palette.error.main,
           },
         },
```

Comment:
Can we move `padding: 0` somewhere up higher so we can see it?

Also, when overriding styles, I prefer overriding the ones we care about. For instance, if you're overriding left and right, explicitly putting `padding-left: 0` is more valuable than `padding: 0` because the all-directions override doesn't give me any information of _what_ you're overriding. It just looks like a reset. It doesn't tell me where the padding came from.

Sometimes, this can have unexpected results, but also, I'm thinking about long-term maintenance. A dev should be able to get in and figure out which were overrides and which were resets.